app/views/catalog/facet.html.erb

```diff
@@ -14,9 +14,9 @@
 <div class="modal-body">
   <div role="region" id="facet_list_header" aria-live="polite">
     <% if @pagination.prefix && @display_facet.index? %>
-      <h4>Topics starting with "<%= @pagination.prefix %>" </h4>
+      <h4><%= facet_field_label(@facet.key) %>s starting with "<%= @pagination.prefix %>" </h4>
```

Comment:
Strings in Ruby actually have a `#pluralize` method. Its WILD!
```
'post'.pluralize             # => "posts"
'octopus'.pluralize          # => "octopi"
'sheep'.pluralize            # => "sheep"
```

Reply:
That's awesome, fixed!